recipes-openxt/xenclient/xenclient-root-ro/init.root-ro

```diff
@@ -314,13 +314,6 @@ unlock_config()
     case "$sig" in
     LUKS)
         if [ -e /boot/system/tpm/enabled ]; then
-            [ "$measured" != 0 ] && {
```

Comment:
Maybe explicitly state in the commit messages that this is okay?  A DRTM boot doesn't need the TXT check since the unseal will fail?

Reply:
Correct, I'll update the commit msg.